test/e2e/service.go

```diff
@@ -52,6 +52,10 @@ const loadBalancerLagTimeout = 2 * time.Minute
 
 // How long to wait for a load balancer to be created/modified.
 //TODO: once support ticket 21807001 is resolved, reduce this timeout back to something reasonable
+//TODO: this timeout is actually used in two distinct senses; once when waiting
```

Comment:
This is not quite right - I used it again after changing the port number because that operation (on GCE) causes a full rebuild of the LB, so is akin to a create operation.